### PR TITLE
Intro to Exercism.io for HW

### PR DESCRIPTION
**Pull request by CheezItMan**
@kariabancroft @PilgrimMemoirs @droberts-ada @Hamled 

Just a doc to get them setup with exercism

### CODE REVIEW

File: exercism-io.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+## Overview
+
+Exercism.io is a set of mini-quests designed to introduce you to a new language, in this case Javascript.  It will present you with problems and a set of test suites to verify that your solution is correct.  Learning by TDD!  It's a great way to explore the language.
+
+## Setup
+
+To install Exercism first install Nodejs (you may have this already installed.
+
+```bash
+$  brew install node
```
Comment (kariabancroft) [inline exercism-io.md]: We can remove this because they will install this to start

**Comment by kariabancroft** [comment]
Looks good - I'd probably live-code the first test together and then allow folks to do the second one in class at the same time.

File: exercism-io.md (hunk 1 above)
Comment (CheezItMan) [reply exercism-io.md]: Will do, although I'll leave it commented in the doc in case we want them installing this on their own before JS starts next time.

**Comment by CheezItMan** [comment]
@kariabancroft That's the intention, probably live-code both hello-world & leap because leap has some features hello-world doesn't (the `modules.exports` component).

**Pull request merged**